realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -411,6 +411,7 @@ private static void initializeSyncedRealm(Realm realm) {
                 realm.setVersion(newVersion);
                 commitChanges = true;
             }
+            schema.close();
```

Comment:
Yes, right now, but `close()` is on the abstract class, and functionality will probably shift between the two classes, so we cannot make any assumptions